source/includes/steps-3.6-upgrade-sharded-cluster.yaml

```diff
@@ -38,38 +38,36 @@ title: "Upgrade the config servers."
 level: 4
 ref: 3.6-upgrade-config-servers
 content: |-
-   If the config servers are replica sets:
-      
    1. Upgrade the :ref:`secondary <replica-set-secondary-members>`
       members of the replica set one at a time:
 
       a. Shut down the secondary :program:`mongod` instance and replace
          the |oldversion| binary with the |newversion| binary.
 
-      #. Start the |newversion| binary with both the ``--configsvr``,
-         ``--port`` options, 
-
-         If the sharded cluster members are run on different hosts or
-         if you wish remote clients to connect to the sharded cluster,
-         you must also specify ``--bind_ip`` (or the
-         :setting:`net.bindIp` setting). For more information, see
-         :ref:`3.6-bind_ip-compatibility`.
+      #. Start the |newversion| binary with the following required
+         options and any previously used optional command line options:
```

Comment:
Which options in the code block are required and which are optional?  As only configsvr and replSet are required.